pages/_includes/statewide-footer.njk

```diff
@@ -42,8 +42,12 @@
       <p class="copyright">Copyright &copy;&nbsp;
         <span class="js-cagov-current-year"></span> <script>document.querySelector('.js-cagov-current-year').innerHTML = new Date().getFullYear();</script>&nbsp;State of California
       </p>
+
+      {% block ppa %}
```

Comment:
Nitpick but can we change this easily to `block par` since it has been renamed?

Reply:
And the line after

Reply:
Yep, update checked in